ckanext/dcat/profiles/euro_dcat_ap_base.py

```diff
@@ -123,7 +123,7 @@ def _parse_dataset_base(self, dataset_dict, dataset_ref):
 
         # Publisher
         publisher = self._publisher(dataset_ref, DCT.publisher)
-        for key in ("uri", "name", "email", "url", "type"):
+        for key in ("uri", "name", "email", "url", "type", "identifier"):
```

Comment:
This will take care of the DCAT -> CKAN mapping, but you also need to add the other way around, CKAN -> DCAT. This is done a bit further down in the file [here](https://github.com/ckan/ckanext-dcat/blob/cb25389f11effefc28165b54de9ec68f61a86898/ckanext/dcat/profiles/euro_dcat_ap_base.py#L370) and [here](https://github.com/ckan/ckanext-dcat/blob/cb25389f11effefc28165b54de9ec68f61a86898/ckanext/dcat/profiles/euro_dcat_ap_base.py#L404).  

Additionally, this will cover the legacy way of defining publisher fields based on namespaced `publisher_*` extra fields, but going forward we want to support scheming objects (`"publisher": {"id": xx, "name": yy, "url": zz}`). This is handled in the `euro_dcat_ap_scheming` profile [here](https://github.com/ckan/ckanext-dcat/blob/cb25389f11effefc28165b54de9ec68f61a86898/ckanext/dcat/profiles/euro_dcat_ap_scheming.py#L169) (for CKAN -> DCAT, the opposite should already be handled by the base profile). Do you mind adding support for the new field there as well? And if you extend [this test](https://github.com/ckan/ckanext-dcat/blob/cb25389f11effefc28165b54de9ec68f61a86898/ckanext/dcat/tests/profiles/dcat_ap_2/test_scheming_support.py#L303) to cover the new field, even better.

@hcvdwerf Let me know if all this makes sense

Reply:
Tnx for the comments. Verry helpful @amercader . Jut commit the changes